### DISCUSSION TITLE
Don't remember recent searches in a private container

### DISCUSSION THREAD

**Discussion by CactusPie** [Floorp-Projects/Floorp#1462] [Feature Request]
I really love the "Private container" feature in Floorp, but there is one small issue with it. When I'm currently using a tab that's opened in a private container, all recent searches (via the address bar) will be remembered - even if I disable **Show recent searches** in settings. Is there any chance this could be disabled? Either by default or via a option in the settings. Much appreciated!

**Comment by ioc2e3** [comment]
> ... ，但它有一個小問題。當我目前使用在私人容器中開啟的選項卡時，所有最近的搜尋（透過網址列）都會被記住 - 即使我在設定中停用**顯示最近的搜尋**。有可能停用此功能嗎？預設或透過設定中的選項。非常感謝！

Agree ...

Hope search record for private container don't be remembered ...

.
